src/Runner/TestSuiteSorter.php

```diff
@@ -30,27 +30,71 @@
      */
     public const ORDER_REVERSED = 2;
 
+    /**
+     * @var int
+     */
+    public const ORDER_DEFECTS_FIRST = 3;
+
+    /**
+     * List of sorting weights for all test result codes. A higher number gives higher priority.
+     */
+    private const DEFECT_SORT_WEIGHT = [
+        BaseTestRunner::STATUS_ERROR      => 6,
+        BaseTestRunner::STATUS_FAILURE    => 5,
+        BaseTestRunner::STATUS_WARNING    => 4,
+        BaseTestRunner::STATUS_INCOMPLETE => 3,
+        BaseTestRunner::STATUS_RISKY      => 2,
+        BaseTestRunner::STATUS_SKIPPED    => 1,
+        BaseTestRunner::STATUS_UNKNOWN    => 0
+    ];
+
+    /**
+     * @var array[string]int Associative array of (string => DEFECT_SORT_WEIGHT) elements
```

Comment:
this syntax is invalid
`array<string, int>` shall be used

Reply:
fixed